package.json

```
@@ -36,6 +38,7 @@
     "@babel/preset-env": "^7.5.4",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.4.4",
+    "@types/react": "^16.9.2",
```

Should this match the version of react pulled in below? Obviously, npm will just grab latest satisfying semver, but it seems like it's safest for these two to match.

```suggestion
    "@types/react": "^16.8.6",
```

---

Great point, I thought the caret in semver would ignore "_minor_._patch_" versions and download the latest `16.x.x` when installed. Do you think it's necessary for us to match those versions?